rdr_service/participant_enums.py

```diff
@@ -714,6 +718,16 @@ class GenomicManifestTypes(messages.Enum):
     CVL_W3 = 7
     AW3_ARRAY = 8
     AW3_WGS = 9
+    AW2F = 10
+
+
+class GenomicContaminationCategory(messages.Enum):
+    UNSET = 0
+    NO_EXTRACT = 1
+    EXTRACT_WGS = 2
+    EXTRACT_BOTH = 3
+    TERMINAL_NO_EXTRACT = 4
```

Comment:
Is this for future use?   (Not yet used in the implementation of the contamination changes?)

Reply:
Yes, this category is more significantly more complicated and is going to be implemented in the upcoming sprint.